packages/autoloader/composer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "automattic/jetpack-autoloader",
+	"description": "Creates a custom autoloader for a plugin or theme.",
+	"type": "composer-plugin",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"composer-plugin-api": "^1.1"
+	},
+	"version": "1.0.0",
```

Comment:
So yeah, let's remove this again, after we change the stability constraint in the main composer.json to `@dev`.
```suggestion
```